flink-connectors/flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/internals/fanout/FanOutProperties.java

```diff
@@ -0,0 +1,377 @@
+/*
```

Comment:
@xiaolong-sn can you please move this class so it is consistent with the Polling configuration to:
- `org.apache.flink.streaming.connectors.kinesis.internals.publisher.fanout.FanOutPublisherConfiguration`

Reply:
OK